### PR TITLE
reactive form widgets example and readme

### PR DESCRIPTION
**Pull request by vasilich6107**
#71

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,22 @@ dependencies:
 
 Then run the command `flutter packages get` on the console.
 
+## Additional reactive fields
```
Comment (joanpablo) [inline README.md:38]: While *fields* is right, I prefer *field widgets*:
Additional reactive field widgets

File: README.md [hunk 2]
```diff
@@ -35,6 +35,22 @@ dependencies:
...
+
+We are trying to keep `reactive_forms` from bloating with third party dependencies this is why there is 
+a separate library [`reactive_forms_widgets`](https://pub.dev/packages/reactive_forms_widgets) which provides
+wide variety of fields which could be helpful.
```
Comment (joanpablo) [inline README.md:42]: "a variety of field widgets"

File: README.md [hunk 3]
```diff
@@ -1110,6 +1126,13 @@ Widget build(BuildContext context) {
 - ReactiveDatePicker
 - ReactiveTimePicker
 
+## Reactive form widgets [library](https://pub.dev/packages/reactive_forms_widgets)
+
+- ReactiveDateTimePicker
+- ReactiveDropdownSearch
+- ReactiveTouchSpin
+- ReactiveSegmentedControl
+
```
Comment (joanpablo) [inline README.md:1135]: ## Advanced Reactive Field Widgets

**Review by joanpablo — COMMENTED** [review]
Once Again, Great work @vasilich6107.
I left you some comments before merging the request.

**Comment by vasilich6107** [comment]
Fixed

**Pull request merged**